src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -93,15 +93,34 @@ const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
       {(() => {
         if (!updateMode) {
           return (
-            <Button
-              mode="strong"
-              onClick={onResendEmail}
+            <div
               css={`
-                margin-top: ${3 * GU}px;
+                display: flex;
+                justify-content: space-between;
+                flex-direction: ${compactMode ? 'column' : 'row'};
+                width: 100%;
               `}
             >
-              Resend verification email
-            </Button>
+              <ActionButton
+                compactMode={compactMode}
+                mode="strong"
+                onClick={onResendEmail}
+                css={`
+                  margin-top: ${3 * GU}px;
```

Comment:
Lets set the `margin-top` at the parent div and remove it from the button styles